modules/admin/proc-install-operator.adoc

```diff
@@ -37,5 +37,6 @@ For enhanced security, deploy the {product} Operator in a dedicated default name
 
 .Additional resources
 
-* For more information about installing {product-short}, see xref:proc-install-rhdh-ocp-operator_admin-rhdh[Deploying {product} on {ocp-short} using the Operator].
+//* For more information about installing {product-short}, see xref:proc-install-rhdh-ocp-operator_{context}[Deploying {product} on {ocp-short} using the Operator].
```

Comment:
We can also update the cross-references in this PR using the ID or the assembly. What do you think?
@Gerry-Forde

Reply:
Linking to the assembly, wherever that makes sense, is a great idea